gateway/api/serializers.py

```diff
@@ -187,3 +187,71 @@ class RuntimeJobSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = RuntimeJob
+
+
+class RunProgramSerializer(serializers.Serializer):
+    """
+    Program serializer for the /run end-point
+    """
+
+    title = serializers.CharField(max_length=255, allow_blank=False)
+    entrypoint = serializers.CharField(max_length=255)
+    artifact = serializers.FileField(allow_empty_file=False, use_url=False)
+    dependencies = serializers.CharField(allow_blank=False)
+    arguments = serializers.CharField(allow_blank=False)
+    config = serializers.CharField(allow_blank=False)
```

Comment:
I think this was `JSONField`.  Then you can put back the changes in #1285.

Reply:
We almost write at the same time. Check my comment here, Aki: https://github.com/Qiskit-Extensions/quantum-serverless/pull/1286#issuecomment-2062135522

I changed to `JSONField` but if I change it the I receive an error: "JSON format invalid".